install.sh

```diff
@@ -32,6 +32,7 @@ installDependencies()
         apt -qq install software-properties-common
         add-apt-repository -qq --yes --update ppa:ansible/ansible
         apt -qq -y install ansible
+        apt install etcd-client
```

Comment:
apt -qq -y. This will ensure that confirmations are automatically accepted.

Reply:
May be we can create a new if block and add this line within.

Reply:
added if block